integrations/jest-plugin/src/index.ts

```diff
@@ -18,10 +21,12 @@ type RawSourceMap = ReturnType<typeof transform>["sourceMap"];
 export function process(
   src: string,
   filename: string,
+  options: TransformOptions<Options>,
 ): {code: string; map?: RawSourceMap | string | null} {
   const transforms = getTransforms(filename);
   if (transforms !== null) {
```

Comment:
@alangpierce Re your comment below - Should we change this line to `if (transforms !== null && options.transformerConfig.transforms !== null)`? I can't come up with a case where you want to apply sucrase to other file types than those already covered by `getTransforms`, but :man_shrugging:

Reply:
I think it's best to leave it as `if (transforms !== null)` since .json files should always be skipped, for example. I think `getTransforms` does need to be updated to account for `.mjs`, `.cjs`, `.mts`, and `.cts`, but that can be a separate fix.

Reply:
Agreed